test/Inputs/SmallStringTestUtilities.swift

```diff
@@ -0,0 +1,114 @@
+#if _runtime(_ObjC)
+import Foundation
+#endif
+import StdlibUnittest
+
+extension String: Error {}
+
+func verifySmallString(_ small: _SmallString, _ input: String) {
```

Comment:
nit: This should get the source position of the caller & pass it to `expect*`, or a failure here will be annoying to track down.

Reply:
Done!